### PR TITLE
adding Helium multitasker app v.1.5.1

### PR DESCRIPTION
**Pull request by erichyland**
An app that lets you watch videos over your workspace. I'm sure it will make everyone more productive.
http://jadengeller.github.io/Helium/

### CODE REVIEW

**Comment by vitorgalvao** [comment]
There’s already [another cask with that name](https://github.com/caskroom/homebrew-cask/blob/11d8e1d36fff2f1f700e05e21393539fdad38bb8/Casks/helium.rb). This needs to be renamed to `jadengeller-hellium`.

File: Casks/helium.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask :v1 => 'helium' do
+  version '1.5.1'
+  sha256 '3be70bd4a20b252ecd1e633374145cfab36cc16c20088670a8801cd7e8e004b6'
+
+  url 'https://github.com/JadenGeller/Helium/releases/download/v1.5.1/Helium.app.zip'
```
Comment (vitorgalvao) [inline Casks/helium.rb]: As long as we’re making changes, please change this to `url "https://github.com/JadenGeller/Helium/releases/download/v#{version}/Helium.app.zip"` as it’ll make updates easier, and add a `appcast 'https://github.com/JadenGeller/Helium/releases.atom'` line under it.

File: Casks/helium.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  name 'Helium'
+  homepage 'http://jadengeller.github.io/Helium/'
+  license :unknown    # todo: change license and remove this comment; ':unknown' is a machine-generated placeholder
```
Comment (vitorgalvao) [inline Casks/helium.rb]: `license :mit`.

**Comment by vitorgalvao** [comment]
There are a bunch of conflicts here; you need to pull changes from upstream before making the changes. I’ll merge this in a different commit, with the corrections. Thank you for the submission.

**Comment by erichyland** [comment]
This is actually first public pull request. Sorry for any newb mistakes. Thanks for Cask I love it.

**Pull request closed without merging**